### PR TITLE
Access Control CS Grammar Improvements

### PR DESCRIPTION
**Pull request by sheerazali19**
updated the document looked for spelling mistakes and grammatical errors. also tried to make the document more clear to read.

Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double check the file for other mistakes in order to fix all the issue on the current cheat sheet.

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [ ] All the markdown files do not raise any validation policy violation, see policy [here](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
- [ ] All your assets are stored in the **assets** folder.
- [ ] All the images used are in the **PNG** format.
- [ ] Any references to website have been formatted as [TEXT](URL)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).

If your PR is related to an issue. Please end your PR text with the following line:

This PR covers issue #<insert number here>.

Thank you again for your contribution :smiley:

### CODE REVIEW

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 ## What is Access Control / Authorization?
 
-Authorization is the process where requests to access a particular resource should be granted or denied. It should be noted that authorization is not equivalent to authentication - as these terms and their definitions are frequently confused. Authentication is providing and validating identity. Authorization includes the execution rules that determines what functionality and data the user (or Principal) may access, ensuring the proper allocation of access rights after authentication is successful.
+Authorization is the process where requests to access a particular resource should be granted or denied. It should be noted that authorization is not equivalent to authentication - as these terms and their definitions are frequently confused. Authentication is providing and validating identity. The authorization includes the execution rules that determine what functionality and data the user (or Principal) may access, ensuring the proper allocation of access rights after authentication is successful.
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md]: The first change is incorrect - it should just be `Authorization`. The second one is valid though.

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 2]
```diff
@@ -45,22 +45,22 @@ The areas of caution while using RBAC are:
...
 
-A DAC framework can provide web application security administrators with the ability to implement fine grained access control. This model can be a basis for data based access control implementation
+A DAC framework can provide web application security administrators with the ability to implement fine-grained access control. This model can be a basis for data-based access control implementation
 
 The advantages of using this model are:
 
 - Easy to use
 - Easy to administer
-- Aligns to the principle of least privileges.
+- It aligns to the principle of least privileges.
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md]: This is not inconsistent with the other elements of the list. They should either all be as they are (Easy, Aligns, Object), or all like this (It is easy, it aligns, the object has..)

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
 ## What is Access Control / Authorization?
 
-Authorization is the process where requests to access a particular resource should be granted or denied. It should be noted that authorization is not equivalent to authentication - as these terms and their definitions are frequently confused. Authentication is providing and validating identity. Authorization includes the execution rules that determines what functionality and data the user (or Principal) may access, ensuring the proper allocation of access rights after authentication is successful.
+Authorization is the process where requests to access a particular resource should be granted or denied. It should be noted that authorization is not equivalent to authentication - as these terms and their definitions are frequently confused. Authentication is providing and validating identity.Authorization includes the execution rules that determine what functionality and data the user (or Principal) may access, ensuring the proper allocation of access rights after authentication is successful.
```
Comment (ThunderSon) [inline cheatsheets/Access_Control_Cheat_Sheet.md]: You removed the white space before Authorization. Kindly tend to that

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 4]
```diff
@@ -14,11 +14,11 @@ Why do we need an access control policy for web development?
...
 
-The process of defining roles is usually based on analyzing the fundamental goals and structure of an organization and is usually linked to the security policy. For instance, in a medical organization, the different roles of users may include those such as doctor, nurse, attendant, nurse, patients, etc. Obviously, these members require different levels of access in order to perform their functions, but also the types of web transactions and their allowed context vary greatly depending on the security policy and any relevant regulations (HIPAA, Gramm-Leach-Bliley, etc.).
+The process of defining roles is usually based on analyzing the fundamental goals and structure of an organization and is usually linked to the security policy. For instance, in a medical organization, the different roles of users may include those such as a doctor, nurse, attendant, nurse, patients, etc. Obviously, these members require different levels of access in order to perform their functions, but also the types of web transactions and their allowed context vary greatly depending on the security policy and any relevant regulations (HIPAA, Gramm-Leach-Bliley, etc.).
```
Comment (ThunderSon) [inline cheatsheets/Access_Control_Cheat_Sheet.md]: Can you remove the double nurse mentioning? `doctor, nurse, attendant, nurse, patients,`

**Pull request merged**